pkg/utils/utils.go

```diff
@@ -565,8 +569,19 @@ func setVfAdminMac(vfAddr string, pfLink netlink.Link) error {
 	}
 	vfLink, err := vfIsReady(vfAddr)
 	if err != nil {
-		glog.Errorf("setVfAdminMac(): VF link is not ready for device %+v %q", vfAddr, err)
-		return err
+		err = RebindVfToDefaultDriver(vfAddr)
```

Comment:
Thinking on this more, this is a surprising place to find a WA.

here we basically just do `ip link set <pf> vf <idx> mac <macaddr>`

the reason i assume its here is because `vfIsReady` was put here in commit : https://github.com/k8snetworkplumbingwg/sriov-network-operator/commit/eb477039f94b4ccd0b5d9cb076e1b92ffd888a43

I wonder if a better approach is to roll the WA logic into `VfIsReady` and call it from `configSriovDevices`

essentially after we create the VFs, we wait for them to be ready then move on with our business.

as the issue only happens on Intel, you dont need to worry about a rebind later on that might hit the issue.
which happens only with rdma.(also as the WA is applied today it will not solve it)